vagrant/centos8.ks

```diff
@@ -0,0 +1,147 @@
+#url --mirrorlist=http://mirrorlist.centos.org/?release=8&arch=x86_64&repo=BaseOS&infra=stock
+#repo --name=AppStream --mirrorlist=http://mirrorlist.centos.org/?release=8&arch=x86_64&repo=AppStream&infra=stock
+install
+text
+keyboard us
+lang en_US.UTF-8
+skipx
+network --bootproto dhcp
+rootpw --plaintext vagrant
+firewall --disabled
+authconfig --enableshadow --enablemd5
+selinux --enforcing
+timezone --utc UTC
+services --enabled=vmtoolsd,NetworkManager,sshd
+# The biosdevname and ifnames options ensure we get "eth0" as our interface
+# even in environments like virtualbox that emulate a real NW card
+bootloader --timeout=1 --append="no_timer_check console=tty0 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 elevator=noop"
+zerombr
+clearpart --all --drives=vda
+part / --fstype=xfs --asprimary --size=1024 --grow --ondisk=vda
+
+user --name=vagrant --password=vagrant
+
+reboot
+
+%packages --instLangs=en
+bash-completion
+man-pages
+bzip2
+rsync
+nfs-utils
+cifs-utils
+chrony
+yum-utils
+hyperv-daemons
+open-vm-tools
+# Vagrant boxes aren't normally visible, no need for Plymouth
+-plymouth
+# Microcode updates cannot work in a VM
+-microcode_ctl
+# Firmware packages are not needed in a VM
+-iwl100-firmware
+-iwl1000-firmware
+-iwl105-firmware
+-iwl135-firmware
+-iwl2000-firmware
+-iwl2030-firmware
+-iwl3160-firmware
+-iwl3945-firmware
+-iwl4965-firmware
+-iwl5000-firmware
+-iwl5150-firmware
+-iwl6000-firmware
+-iwl6000g2a-firmware
+-iwl6000g2b-firmware
+-iwl6050-firmware
+-iwl7260-firmware
+-libertas-usb8388-firmware
+#-linux-firmware # Required by kernel-core, can't remove
+# Don't build rescue initramfs
+-dracut-config-rescue
+# Disable kdump
+# -kexec-tools
+%end
+
+%post
+# configure swap to a file (fallocate doesn't work with c7 xfs)
+dd if=/dev/zero of=/swapfile bs=1M count=2048
```

Comment:
'fallocate -l 2G /swapfile' is better because it generates less IO during installation, but it didn't work with the old kernel in CentOS 7; we can use that again in CentOS 8